search/search_params.hpp

```diff
@@ -37,16 +35,26 @@ struct SearchParams
   std::string m_query;
   std::string m_inputLocale;
 
-  boost::optional<ms::LatLon> m_position;
+  boost::optional<m2::PointD> m_position;
   m2::RectD m_viewport;
 
+  size_t m_maxNumResults = kDefaultNumResultsEverywhere;
+
   // A minimum distance between search results in meters, needed for
   // pre-ranking of viewport search results.
   double m_minDistanceOnMapBetweenResults = 0.0;
 
   Mode m_mode = Mode::Everywhere;
   bool m_forceSearch = false;
-  bool m_suggestsEnabled = true;
+
+  // Is it needed to generate search suggests.
```

Comment:
Please write also what m_forceSearch is needed for.

Reply:
Done